dbms/src/Flash/Mpp/MPPTask.cpp

```diff
@@ -336,6 +348,7 @@ void MPPTask::preprocess()
 void MPPTask::runImpl()
 {
     CPUAffinityManager::getInstance().bindSelfQueryThread();
+    current_memory_tracker = process_list_entry->get().getMemoryTrackerPtr().get();
```

Comment:
is it necessary? since `newThreadManager()->scheduleThenDetach` will propgate memTracker into `runImpl`

Reply:
Oh, yes, it is not necessary.

Reply:
Maybe change it to RUNTIME_ASSERT

Reply:
Done